deploy/charts/openebs-monitoring/dashboards/cStor/cstor-volumes.json

```diff
@@ -1034,11 +1069,38 @@
         "skipUrlSync": false,
         "type": "datasource"
       },
+      {
+        "allValue": null,
+        "current": {},
+        "datasource": null,
+        "definition": "label_values(openebs_size_of_volume,openebs_cstor_label)",
```

Comment:
Remove this definition. Make it an empty string.

Reply:
There is another field `description` which is used for describing a variable. 
But when you will create the variable from the grafana then `definition` will have the same value as that of the query.

Reply:
Cool